src/AltinnCore/Common/Languages/ini/en.ini

```diff
@@ -33,6 +33,18 @@ filter_your_services = Your services
 new_service = new service
 sorte_services = Sort
 last_changed_service = Changed:
+new_service_header = New service
+service_owner = Serviceowner
+service_owner_description = The service will be visible from the serviceowners dashboard. All members of the serviceowners organization will be able to see and edit the service.
+service_name = Servicename
+service_name_description = This name will be shown to the end user and schould be descriptive to what the service is about. This name can be changed later.
+service_saved_name = Repository name
+service_saved_name_descripyion = This name will only be used on a technical level for creating the service. The name need to start with a letter and can only alphanumeric, dash ('-'), underscore ('_') and dot ('.') characters. This name cannot be changed at a later time.
```

Comment:
The name needs...
...and can only contain alphanumeric...

Reply:
Can the service name actually contain underscore? @TheTechArch? Or will this break stuff? In StartServiceModel.cs (AltinnCore.ServiceLibrary) the service id (org + service name) is underscore separated, so if a service name has underscore in it I think start service will break.

Reply:
I tested that the code works with underscore in service name, and the code you are refering to can easily ble changed so that org and service is separated by two underscrores instead of one
It looks like runtime is not working at all so its hard to test if it acctually breaks or what happens